stdlib/Random/src/deprecated.jl

```diff
@@ -37,3 +37,6 @@ end
 
 # PR #25668
 @deprecate RandomDevice(unlimited::Bool) RandomDevice(; unlimited=unlimited)
+
+# PR #28295
+@deprecate srand Random.seed false
```

Comment:
I was actually wrong about `false`; that avoids exporting `srand`, not `Random.seed`. What you had originally was correct. Sorry about that!